### PR TITLE
Embedded graphics 7.0 update

### PR DESCRIPTION
**Pull request by jacobrosenthal**
https://github.com/embedded-graphics/embedded-graphics/blob/0ba2b553992e4b1188516a2a5079508769fcd571/MIGRATING-0.6-0.7.md

I only tested pygamer on hardware. BSP owners should confirm thiers.

The big change was some fonts that dont exist anymore:

> The largest available new font is FONT_10X20, which is significantly smaller than the old Font24x32. Larger fonts are available in [external crates](https://github.com/embedded-graphics/embedded-graphics#additional-functions-provided-by-external-crates).

wiolite used a Font12x16 which i replaced with a FONT_9X15, and a Font24x32 which I replaced with a FONT_10X20. I'm sure they look wrong now and maybe someone wants to pick something else.

### CODE REVIEW

**Review by sajattack — COMMENTED** [review]
I'm guessing this file wasn't intended to be included in this pr, but I don't mind it tbh.

File: boards/edgebadge/.embed.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[default.probe]
```
Comment (sajattack) [inline boards/edgebadge/.embed.toml:1]: Sorry, I didn't click the right spot evidently.

**Comment by sajattack** [comment]
Looks like rustfmt hates your doc comment or something?

**Comment by sajattack** [comment]
@jessebraham @twitchyliquid64 Do either of you want to try wio terminal before I merge this?

**Comment by bradleyharden** [comment]
@sajattack, I don't know that we're going to get any additional testing. Should we merge anyway?

**Comment by sajattack** [comment]
Sure.

**Pull request merged**